### DISCUSSION TITLE
Monitor and/or log client user logged in activity and free disk space

### DISCUSSION THREAD

**Discussion by gajotnt** [Ylianst/MeshCentral#6623] [Ideas]
I work at a school and we use meshcentral to monitor and give support to students and teachers/workers.

Since Meshcentral already knows what user is logged in and logs the powerstate of the computer, cant it also log the user that was using at that time?

This would be very usefull to know what student used the computer at a specific time. Kids sometimes do bad stuff and its good to know who used the PC in a specific time interval.

Also in the Devices list view, since meshcentral already knows the disk usage, would be awesome to be able to add a column to see available disk space of each device. That way i could tell if a computer needs a disk cleanup before teachers complain. 

I know there are other monitoring softwares that can do this, but why add one more, if meshcentral can be able do it?

![Screenshot 2024-12-22 142101](https://github.com/user-attachments/assets/8363d114-dae3-40d6-8db8-d94675e83f3b)
![Screenshot 2024-12-22 142317](https://github.com/user-attachments/assets/26898613-d219-4d8b-a603-3220a80751eb)

**Comment by si458** [comment]
both of those things are feature requests and we would need to implement them
you can create an github issue selecting enhancement for EACH of the features you want adding (seperate them plz!)

meshcentral was always designed as like a remote support software to help people
it was never designed as a remote management software (even tho the tag on github says it is)

the Active Users also doesnt always work correctly but it does try its best to say whos logged in when a user logs in
but even now i have an issue from time to time where it thinks a users logged in but actually the is no user logged in at all!

as for disk capacity, that could possibly be done, but what is devces have multiple drives? then it becomes an issue

**Reply by gajotnt** [reply]
Done, thanks for the insigh for the possible issues
